lib/backend.php

```diff
@@ -136,7 +136,7 @@ public function getListing($FOLDER, $showdel) {
 		// Get the listing from the database
 		$requery = false;
 		$uid = \OCP\User::getUser();
-		$query = "SELECT id, name, grouping, mtime, deleted FROM *PREFIX*ownnote WHERE uid=? ORDER BY name";
+		$query = "SELECT id, name, `grouping`, mtime, deleted FROM *PREFIX*ownnote WHERE uid=? ORDER BY name";
```

Comment:
I'm curious about this change. I don't see this needed based on column type and MySQL 8 release/change notes so am curious about documentation as to why these backticks are required now.

Reply:
Please take a look at the link I provided in the PR description (https://dev.mysql.com/doc/refman/8.0/en/keywords.html):

> GROUPING (R); added in 8.0.1 (reserved)

Grouping is a reserved keyword starting with MySQL 8.0.1, hence it's unquoted usage in queries will cause an error.
Example output from a MySQL database with ownnote installed:
```
mysql> SELECT id, name, grouping, mtime, deleted FROM oc_ownnote ORDER BY name LIMIT 1;
ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ', mtime, deleted FROM oc_ownnote ORDER BY name LIMIT 1' at line 1
mysql> SELECT id, name, `grouping`, mtime, deleted FROM oc_ownnote ORDER BY name LIMIT 1;
+----+------------+----------+------------+---------+
| id | name       | grouping | mtime      | deleted |
+----+------------+----------+------------+---------+
| 34 | Abrechnung |          | 1587824928 |       0 |
+----+------------+----------+------------+---------+
1 row in set (0.00 sec)
```

Reply:
@mh0rst I apologize. I did not see that GROUPING had become reserved. Thank you for educating me.